frontend/front/src/pages/Public/Pages/SurveyPage/SurveyPage.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Button, Container, Stack, TextField } from "@mui/material";
+import { HeatPumpAddressField } from "./HeatPumpAddressField";
+import { HeatPumpNameField } from "./HeatPumpNameField";
+import { HeatPumpDropdown } from "./HeatPumpDropdown";
+
+const DEFAULT_FORM = {
+  heatingSystem: "",
+  name: { first: "", last: "" },
+  isHomeowner: "",
+  phoneNumber: "",
+  contactTime: "",
+  address: {
+    street: "",
+    aptNumber: "",
+    city: "",
+    zipCode: "",
+  },
+};
+
+export const SurveyPage = () => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    control,
+    formState: { errors },
+  } = useForm({ defaultValues: DEFAULT_FORM });
+
+  const onSubmit = (data) => {
+    // TODO: connect to back-end
+    alert(JSON.stringify(data, null, 4));
+  };
+
+  return (
+    <Container>
+      <Stack direction="column" alignItems="center" justifyContent="center">
+        <h1>TAKE THE SURVEY</h1>
+        <p>
+          Fill out this form to record your interest in installing a heat pump
+          for your home.
+        </p>
+        <p>
+          An ULHPA representative will contact you with more information about
+          the installation process.
+        </p>
+      </Stack>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Stack spacing={5} mb={5} mt={5}>
+          <HeatPumpDropdown
+            name="heatingSystem"
+            control={control}
+            label="Current Heating/Cooling System"
+            options={[
+              { label: "Boiler System", value: "Boiler System" },
+              { label: "Gas", value: "Gas" },
+              { label: "Other", value: "Other" },
+            ]}
+          />
+
+          <HeatPumpNameField control={control} />
+
+          <HeatPumpDropdown
+            name="isHomeowner"
+            control={control}
+            label="Are you the homeowner?"
+            options={[
+              { label: "Yes", value: "true" },
+              { label: "No", value: "false" },
+            ]}
+          />
+
+          <TextField
```

Comment:
I believe the correct way to integrate MUI interface with react hook forms.
https://react-hook-form.com/get-started/#IntegratingwithUIlibraries
Your way seems to work but I think it's better if we follow the same pattern incase there is some problem down the road. It looks like you did this in the HeatPumpAddresField so copy that pattern.

Reply:
done